src/content/structured/get-started/releases-versioning.mdx

```diff
@@ -12,7 +12,7 @@ contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/struc
 
```

Comment:
Make sure to update the last updated date so that users know this information is new.